### PR TITLE
Add NUGET_API_KEY

### PR DESCRIPTION
**Pull request by heurtematte**
Related to https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/4244

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by Phil91 — APPROVED** [review]

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
We will soon deploy the app version of otterdog where more things are automated, see https://github.com/adoptium/.eclipsefdn/pull/43 as an example.

The next thing we want to implement is an auto-merge feature where the project can define in an additional config file who is required for approval and once the approvals are received, it gets automerged and applied.

**Pull request merged**
